src/game/rtsutils.h

```diff
@@ -89,6 +89,17 @@ template<int a, int b, int c, int d> struct FourCC
 #endif
 };
 
+inline uint32_t FourCC_By_String(const char *s)
+{
+    if (strnlen(s, 5) > 4)
```

Comment:
This will fail if s is smaller than 4, which is a valid input for things like "EAR\0" -> length of 3.

I suggest to create a `char buffer[4]`, init to 0, copy `s` into it for a maximum of 4 chars, and then convert that to the `uint32_t`.

This way `s` can be any length and this will work.